src/sched/futex.c

```diff
@@ -243,7 +237,7 @@ static int futex_wake(int* uaddr, unsigned int num, uint32_t bitset)
 
     SLIST_FOREACH_SAFE(fq, &futex_queues, entries, tmp)
     {
-        if (fq->futex_key == futex_key && fq->futex_bitset & bitset && w < num)
+        if (fq->futex_key == futex_key && w < num)
```

Comment:
This line makes me sad because it makes me try to remember C operator precedence rules.  Not essential for this PR, but some brackets would be nice.